astro/src/content/docs/get-started/core-concepts/applications.mdx

```diff
@@ -46,6 +46,29 @@ Below is a visual reminder of the relationships between FusionAuth's primary cor
 
 <img src="/img/docs/get-started/core-concepts/core-concepts-relationships-applications.png" alt="Diagram showing Applications used within FusionAuth" />
 
+## Universal Applications
+A universal application is an application that exists for all tenants, rather than belonging to a single tenant. Universal applications are useful when you need to make the same application available to users across many or all tenants.
+
+A universal application uses the same application configuration across all tenants, including its client Id, client credentials, and more. This makes configuration management simpler, allowing you to share configuration across all uses of the application.
+
+### Universal Application Registrations
+Users from any tenant can have a registration to a universal application, and registrations work as they would for normal applications.
+
+### Universal Applications and Tenants
+Becuase a universal application does not belong to a single tenant, you will generally have to supply a tenant Id when interacting with a universal application. This is true during authentication and other user-specific workflows.  The supplied tenant Id tells FusionAuth which tenant to look a user up in, to retrieve theme information from, and more.
+
+The tenant Id is typically provided on a `tenantId` query parameter, unless otherwise specified.
+
+A universal application "inherits" settings from the tenant it is being associated with at runtime. This means that you can customize the behavior of a universal application per tenant by using different tenant configuration.
+
+Note that for tenant configuration settings that can be overridden at the application level, if you supply an application-level override, that setting will be used for all tenants. If this is not what you want, do not set a value in the universal application's configuration and let the application fall back to what is defined in the tenant.
+
+### Universal Applications and Tenant-scoped API Keys
+A tenant-scoped API key cannot make changes that could be seen or otherwise affect other tenants than the one it is scoped to. Because universal applications span all tenants, you cannot use a tenant-scoped API key with a universal application via the [Application API](/docs/apis/applications). You can, however, use a tenant-scoped API key to interact with things like registrations and refresh tokens, as these are confined to a user within a tenant.
+
```

Comment:
We should add a section here on self-serve, mainly to indicate that to use the self serve pages with a universal application a tenantId must be supplied